easybuild/easyconfigs/s/SSN/SSN-1.1.14-foss-2019a-R-3.6.0.eb

```diff
@@ -0,0 +1,36 @@
+#
```

Comment:
nitpicking: any particular reason for this?

Reply:
ah, if there's a comment on the very first line of an `.eb` file, vim will detect it as "conf" filetype and it's nicely highlighted.

probably it makes more sense to leave the usual comment "This is an easybuild recipy..."